txlib/api/base.py

```diff
@@ -332,6 +332,19 @@ def _handle_wrong_field(cls, field_name, field_type):
         _logger.error(msg)
         raise AttributeError(msg)
 
+    @staticmethod
+    def is_binary(data):
+        """Check if provided data can be handled as binary.
+
+        The method assumes that if the received data is not of str and dict
+        type it can be handled as binary.
+
+        :param data: data to check
+        :return: True if data can be handled as binary, False otherwise
+        :rtype: boolean
+        """
+        return not isinstance(data, str) and not isinstance(data, dict)
```

Comment:
I don't think we need a dedicated method for this.

After offline discussions, it turns out that `isinstance(data, dict)` was added so that `TestTranslationModel.test_put_translation()` doesn't break. The reason that it broke, though, is that the content being sent to `Translation.save()` was `content = {'Master_key': 'τεστ'}`.

This doesn't make sense, as content should always be either a string or bytes. Therefore, the check can become just `not isinstance(data, str)` which doesn't warrantee a special method.

Please check the other files for my actual recommendation. As far as the `is_binary()` method is concerned, let's remove it altogether.